themes/default/content/blog/amazing-performance/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Amazing Performance"
+date: 2022-12-01T07:00:00-05:00
+meta_desc: "Introducing the Amazing Performance initiative, and an overview of performance tooling we've developed for Pulumi."
+meta_image: meta.png
+
+authors:
+    - robbie-mckinstry
+
+tags:
+    - performance
+    - platform
+    - engineering
+---
+
+This is the first post in a series about performance optimizations we've made to the Pulumi CLI. Over the last six months at Pulumi, the Platform Team has been working on a project we call "Amazing Performance." Amazing Performance is a new initiative to improve the throughput and latency of the Pulumi CLI not only for power users but for everyone. By the end of June 2022, we assembled a list of issues containing both high-value improvements requiring a sizable investment and low-hanging fruit for quick wins. The full list, including the items we have yet to tackle, is contained in [a tracking issue on GitHub](https://github.com/pulumi/home/issues/1499). This blog series will cover the highlights.
```

Comment:
The tracking issue is private within pulumi. I don't think we should link to it.

Reply:
@t0yv0 also pointed this out. You're right, we shouldn't link to issues in Home.
Maybe about a month back, I talked with @mikhailshilkov about whether we could exfiltrate the issue into the public space. I opened https://github.com/pulumi/pulumi/issues/11598, which is essentially a forward-looking version of the private issue, which we intend to close as it was specifically scoped to Q4 2022.